client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -174,14 +174,16 @@
             <button
               v-if="hasPermission('area_admin_boilerplates')"
               :class="prefixClass('menubar__button')"
+              data-cy="segmentEditor:boilerplate"
               type="button"
               v-tooltip="Translator.trans('boilerplate.insert')"
               @click.stop="openBoilerPlate">
               <i :class="prefixClass('fa fa-puzzle-piece')" />
             </button>
             <button
-              v-if="asyncComponents"
+              v-if="asyncComponents.length > 1"
```

Comment:
Without diving deeper into the code: Shouldn't this be 
```suggestion
              v-if="asyncComponents.length >= 1"
```
?

Reply:
Yeah, of course, thanks! I changed it to `.length > 0` in [ec809f8](https://github.com/demos-europe/demosplan-core/pull/3734/commits/ec809f89929238cbf83d6bc778639f52158d7247)